packages/components/src/Form/src/Form.tsx

```diff
@@ -51,6 +43,30 @@ export interface FormProps extends StyledComponentProps<RACFormProps> {
     size?: ResponsiveProp<FieldSize>;
 }
 
+export interface FormProps extends StyledComponentProps<RACFormProps>, FormStyleProps {}
+
+export const FormStyleContext = createContext<FormStyleProps | null>(null);
```

Comment:
Why do we have 2 context, while v2  only has 1? wouldn't it be simpler to just have 1? having 2 means we might add a prop on FormContext, and when using useFormProps, it wouldn't pick up the property

Reply:
resolved. we now have one form context that using the FormStyleProps.